bloom-instance/s3.tf

```diff
@@ -1,3 +1,30 @@
-resource "aws_s3_bucket" "user_upload_bucket" {
-  bucket_prefix = "${var.name_prefix}-user-uploads"
+resource "aws_s3_bucket" "public_uploads" {
+  bucket_prefix = "${var.name_prefix}-public-uploads"
+}
+
+resource "aws_s3_bucket" "secure_uploads" {
+  bucket_prefix = "${var.name_prefix}-secure-uploads"
+}
+
+resource "aws_s3_bucket" "static_content" {
+  bucket_prefix = "${var.name_prefix}-static-content"
+}
+
+# Allow anyone to read from the public uploads bucket
+resource "aws_s3_bucket_policy" "public_uploads" {
+  bucket = aws_s3_bucket.bucket.id
```

Comment:
Should this be aws_s3_bucket.public_uploads.id instead since this policy is for the public uploads bucket?

Reply:
Yep, you're right.  Good catch